src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -655,5 +655,10 @@ public override IOperation VisitReDimClause(IReDimClauseOperation operation, obj
         {
             return new ReDimClauseOperation(Visit(operation.Operand), VisitArray(operation.DimensionSizes), ((Operation)operation).OwningSemanticModel, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
+
+        public override IOperation VisitUsingDeclaration(IUsingDeclarationOperation operation, object argument)
+        {
+            return new UsingDeclarationOperation(operation.DeclarationGroup, operation.IsAsynchronous, ((Operation)operation).OwningSemanticModel, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
```

Comment:
>operation.DeclarationGroup [](start = 49, length = 26)

Shouldn't we visit this?

Reply:
This is why I want to finish my generator project, to get rid of the auto-generated parts of this class.

---
In reply to: [340878779](https://github.com/dotnet/roslyn/pull/39576#discussion_r340878779) [](ancestors = 340878779)